lib/kitchen/verifier/runtests.rb

```diff
@@ -32,7 +33,7 @@ def call(state)
           File.join(root_path, config[:testingdir], '/tests/runtests.py'),
           '--sysinfo',
           '--output-columns=80',
-          (config[:windows] && config[:tests].empty? ? "--names-file=#{root_path}\\testing\\tests\\whitelist.txt" : ''),
+          (config[:names_file] && config[:tests].empty? ? "--names-file=#{config[:names_file]}" : ''),
```

Comment:
can we just add this as another option instead of removing windows?

Also, if we do choose to remove windows, we should remove it above as well.

I would also be ok with checking if windows is true, and names_file is not specified, and just setting `config[:names_file]` to the default path from the old option.

Reply:
@dwoz any update here?